.github/workflows/support-window.yml

```diff
@@ -0,0 +1,54 @@
+name: Update support window diagram
+on:
+  # When translations are added/removed
+  push:
+    paths: README*.md
+  # Manually, when TypeScript is released
+  # https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          personal <- Sys.getenv("R_LIBS_USER")
+          dir.create(personal, recursive = TRUE)
+          install.packages(c("ggplot2", "svglite"), personal)
+        shell: Rscript {0}
+      - name: Fetch TypeScript versions and release dates from npm
+        run: |
+          npm view --json typescript time |
+            jq '
+              to_entries
+              # Earliest non-pre-release <major>.<minor>
+              | map(
+                (.key |= capture("^(?<version>[0-9]+\\.[0-9]+)\\.[0-9]+$").version)
+                | select(.key)
+              )
+              | unique_by(.key)
+              | from_entries
+            ' > docs/support-window.json
+      - run: git rm --ignore-unmatch 'docs/support-window*.svg'
+      - name: Make SVG diagram
+        run: |
+          export LOCPATH=.locale
+          mkdir "$LOCPATH"
+          for readme in README*.md; do (
+            if [ "$readme" != README.md ]; then
+              LANG=${readme:7:-3}
+              # Use e.g. en_EN if it exists, or the first ${LANG}_*
+              # otherwise
+              locale=/usr/share/i18n/locales/${LANG}_${LANG^^}
+              [ -e "$locale" ] || locale=(/usr/share/i18n/locales/${LANG}_*)
+              localedef --charmap UTF-8 --inputfile "$locale" "$LOCPATH/$LANG"
+            fi
+            jq --raw-output 'to_entries[] | map(values) | @tsv' docs/support-window.json |
+              Rscript scripts/support-window.R
+          ) done
+      - run: git add docs/support-window.*
+      - run: git config user.name .github/workflows/support-window.yml
```

Comment:
Abusing the name seems like a bad idea, should be clear by the commit subject and/or log message.

Reply:
I thought it was a good idea! Maybe not though. :shrug:

git requires a `user.name` and `user.email` and `.github/workflows/support-window.yml` was the most helpful thing I could think to stick in there ... Everything else is dummy values?

Reply:
Here's how it looks in GitHub's UI:
![Screenshot from 2021-09-10 10-55-25](https://user-images.githubusercontent.com/78493/132896927-9acf582c-44ed-4712-b1f8-48f70f0383a2.png)

Reply:
Well, my reasoning is that someone somewhere will write some script looking for some "bot" username (and yes, I know that it's better to use the email etc).  So it's not a strong reason, and the way it looks in github is definitely an improvement...  (I might have used an actual name (as in, an action name rather than the yml filename), but no strong opinions there.)